### PR TITLE
Add specification of Data.ext.segtax

### PR DESCRIPTION
**Pull request by simontrasler**

### CODE REVIEW

File: extensions/community_extensions/segtax.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  ...,
+  "user": {
+    "data": [
+      {
+        "name": "a-data-provider.com",
+        "ext": {
+          "segtax": 3
+        },
+        "segment": [
+          { "id": 1001 },
```
Comment (jdwieland8282) [inline extensions/community_extensions/segtax.md:69]: AdCOM represents `user.data.segment.id` as a string. I think we would all prefer that id be an int.  Both IAB taxonomies (content and audience) represent segments with a "unique id" which as far as I can tell equals a int between 1-1,668.

Comment (simontrasler) [reply extensions/community_extensions/segtax.md:69]: Thanks for pointing this out, `id` is indeed a string. I will fix.

**Comment by patmmccann** [comment]
Is this ready to merge? I'd like to start the process of asking for enumerated 5XX values for a couple custom segmentations we are ready to start putting in bid requests.

**Comment by simontrasler** [comment]
> Is this ready to merge? I'd like to start the process of asking for enumerated 5XX values for a couple custom segmentations we are ready to start putting in bid requests.

Hi @patmmccann -- no, there is one remaining item, to understand where the list will be. I understand it'll be in this repo somewhere, and need the IAB team to confirm.

**Comment by patmmccann** [comment]
appears to be here https://github.com/InteractiveAdvertisingBureau/AdCOM/pull/22/files

**Comment by amitshetty** [comment]
> appears to be here https://github.com/InteractiveAdvertisingBureau/AdCOM/pull/22/files

To be even more precise, here - https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list--category-taxonomies-

**Review by jdwieland8282 — APPROVED** [review]
thanks for adding the table for all available taxonomies

**Pull request merged**